api/envoy/config/metrics/v3/metrics_service.proto

```diff
@@ -55,4 +56,8 @@ message MetricsServiceConfig {
   // and the tag extracted name will be used instead of the full name, which may contain values used by the tag
   // extractor or additional tags added during stats creation.
   bool emit_tags_as_labels = 4;
+
+  // If true, only histogram summary metrics will be emitted to the MetricsService. This is useful
+  // to reduce the size of the metric payloads.
+  bool only_histogram_summary = 5;
```

Comment:
In the PR description you described that if you actually wanted the buckets then you wouldn't want the summary.

Should we add here the ability to send all combinations of buckets, summaries, or nothing?